src/pages/Entity/PunchList/PunchList.tsx

```diff
@@ -38,7 +38,7 @@ const PunchList = (): JSX.Element => {
         return (): void => {
             source.cancel();
         };
-    }, [params.commPkg, params.plant, api]);
+    }, [params.plant, api]);
```

Comment:
search type and entity id from params should be added here instead of the commPkg id